### PR TITLE
Update user guide with new information

### LINKED ISSUE #104: Update and refactor user guide

### PR DESCRIPTION
**Pull request by Hoodini231**
Fixes #104

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -220,7 +260,14 @@ Action | Format, Examples
 **Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
 **Clear** | `clear`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
+**Delete Group** | `deleteGroup [g/GROUP_NAME]` <br> e.g., `deleteGroup g/studygroup1`
```
Comment (martout2002) [inline docs/UserGuide.md:263]: Good addition to functionality

File: docs/UserGuide.md [hunk 2]
```diff
@@ -168,6 +177,33 @@ Format: `deleteGroup GROUPNAME`
 Examples:
 * `groups` followed by `deleteGroup StudyGroup 1` deletes StudyGroup1
 
+## Import students: `import CSV_FILE_LOCATION`
```
Comment (martout2002) [inline docs/UserGuide.md:180]: Deliberate and sufficient explanation of function import

**Review by martout2002 — COMMENTED** [review]
LGTM, good job

**Review by wenjebs — COMMENTED** [review]
looks alright

File: docs/UserGuide.md [hunk 3]
```diff
@@ -127,20 +131,25 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
-* `find /group Study Group returns all members of the group Study Group.`
+* `find /group StudyGroup returns all members of the group StudyGroup.`
```
Comment (yimjianbing) [inline docs/UserGuide.md:134]: will have to update this since find /group has been refactored out

File: docs/UserGuide.md [hunk 4]
```diff
@@ -168,6 +177,33 @@ Format: `deleteGroup GROUPNAME`
...
+
+## Export students: `export`
+
+Exports all students in GoonBook to a .csv file.
+
+Format: `export`
+
+* Exports all students to fixed location as exported_data.csv
+* Location can be found at `[JAR FILE LOCATION]/exported_data.csv`
+
```
Comment (yimjianbing) [inline docs/UserGuide.md:205]: Nice formating!

**Review by yimjianbing — APPROVED** [review]
LGTM! thank you for your hard work shaun!

**Pull request merged**